dev-docs/source/ReleaseChecklist.rst

```diff
@@ -309,55 +312,69 @@ Once the manual testing has passed (check with the Quality Assurance Manager):
 
   *  Have someone at ISIS sign the Windows binary and upload this manually to Sourceforge
 
   *  Set the default package for each OS to the new version using the information icon
      next to the file list on Sourceforge
 
-*  Upload packages to the GitHub release (essentially for a backup).
-*  Publish the GitHub release. This will create the tag required to generate the DOI.
-*  Update the `download <https://download.mantidproject.org>`__ page,
-   following the instructions
-   `here <https://github.com/mantidproject/download.mantidproject.org>`__. Once the new
-   file in the `releases` directory is pushed Jenkins will publish the new page.
-*  Publish the draft release on GitHub (this will create the tag too).
-*  Kick off the build for ``mantidXY`` on RHEL7 for SNS:
-   https://builds.mantidproject.org/job/release_clean-rhel7/ with suffix
-   ``XY``.
+*  Draft a `new release <https://github.com/mantidproject/mantid/releases>`__ on
+   GitHub. The new tag should be created based off the release branch in the form ``vX.Y.Z``. The
+   description of the new release can be copied from the release notes ``index.rst`` file.
+*  Upload the packages to the GitHub release (essentially for a backup), and then publish it. This
+   will create the tag required to generate the DOI.
+*  Update the `download page <https://download.mantidproject.org>`__ by creating a PR after
+   following the instructions in the `Adding a new release section
+   <https://github.com/mantidproject/download.mantidproject.org#adding-a-new-release>`__. Once the
+   new file in the `releases` directory is merged, Jenkins will publish the new page.
+*  Kick off the build for ``mantidXY`` on RHEL7 for the SNS with ``PACKAGE_SUFFIX`` set to
+   ``XY`` where ``X`` and ``Y`` correspond to the Major and Minor release version numbers:
+   https://builds.mantidproject.org/job/release_clean-rhel7/
 * **ISIS**: If in cycle add a calendar reminder for when the current cycle ends for
   mantid to be updated on IDAaaS and cabin PCs. If out of cycle do this immediately.
 
 **Generate DOI**
 
-This requires that a tag has been created for this release. This is done automatically
-if a new `release <https://github.com/mantidproject/mantid/releases>`__ has been
-created on GitHub.
+This requires that a tag has been created for this release. This tag is created when you draft and
+publish a new `release <https://github.com/mantidproject/mantid/releases>`__ on GitHub.
 
 *  Make sure that you have updated your local copy of git to grab the new tag.
    ``git fetch -p``
 *  If the script below fails you may need to update the authors list and push the
    updates to master. Look for ``authors.py`` in the ``tools/DOI`` directory.
    It does not matter that these are not on the release branch.
 
-``python tools/DOI/doi.py  --username=_____  X.Y.Z``
+.. code-block:: bash
+
+    python tools/DOI/doi.py --username=[username] [X.Y.Z]
+    python tools/DOI/doi.py --username="doi.username" 6.1.0
```

Comment:
`python tools/DOI/doi.py --username="doi.username" 6.1.0`

Should be along the lines of:
`python tools/DOI/doi.py --username="doi.username" <release_version>`

Reply:
I put the `6.1.0` in here to make it easy to understand the format of the `<release_version>`. For instance it was difficult to know at first glance if it was expecting `"6.1.0"` or `v6.1.0` or `6.1` or some other format.